### PR TITLE
Example use 'conditional-layout' on product page with product unavail…

### PR DESCRIPTION
**Pull request by carlosviniciusananias**
**What problem is this solving?**
This PR adds an example of using 'conditional layout' to the product page.

**How to test it?**
Just access the page for any [unavailable product](https://cassol.myvtex.com/Ar-Condicionado-Split-Inverter-Philco-18-000-BTUs-Quente-Frio-IQFM9-220V/p).

**A small example**

image

**How does this PR make you feel? 🔗**

image

### CODE REVIEW

File: store/blocks/pdp/product.jsonc [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "store.product": {
     "children": [
       "flex-layout.row#product-breadcrumb",
-      "flex-layout.row#product-main",
+      "condition-layout.product#unavailable",
```
Comment (kaisermann) [inline store/blocks/pdp/product.jsonc:5]: Nit: I'd change the id to `#availability`

File: store/blocks/pdp/product.jsonc [hunk 2]
```diff
@@ -33,6 +33,17 @@
     },
     "children": ["product-description"]
   },
+  "condition-layout.product#unavailable": {
+    "props": {
+      "conditions": [
+        {
+        "subject": "isProductAvailable"
```
Comment (kaisermann) [inline store/blocks/pdp/product.jsonc:40]:
```suggestion
          "subject": "isProductAvailable"
```

**Review by kaisermann — CHANGES_REQUESTED** [review]

File: store/blocks/pdp/product.jsonc (hunk 1 above)
Comment (carlosviniciusananias) [reply store/blocks/pdp/product.jsonc:5]: Hi @kaisermann , done

**Review by kaisermann — APPROVED** [review]

**Comment by carlosviniciusananias** [comment]
Is it worth doing a pr demonstrating a "product unavailable" message directly on the shelves (home / category)?
I have something ready that I just used here, a good example for other developers. What do you think @kaisermann ?

**Review by carlosviniciusananias — COMMENTED** [review]
Manifest.json has been adjusted.
I cannot understand the danger error. How to fix?

**Review by carlosviniciusananias — COMMENTED** [review]
Add "vtex.condition-layout":  "2.x" to manifest.json

File: manifest.json [hunk 3]
```diff
@@ -60,6 +60,10 @@
     "vtex.store-form": "0.x",
     "vtex.product-highlights": "2.x",
     "vtex.product-specifications": "1.x",
+    "vtex.tab-layout": "0.x",
+    "vtex.responsive-layout": "0.x",
+    "vtex.condition-layout": "2.x",
+    "vtex.css-handles": "1.x"
```
Comment (victorhmp) [inline manifest.json:66]:
```suggestion
    "vtex.css-handles": "1.x",
```

**Review by victorhmp — CHANGES_REQUESTED** [review]
Thanks for the PR @carlosviniciusananias ! This will be very useful!
Could you please just rebase it?

File: manifest.json (hunk 3 above)
Comment (carlosviniciusananias) [reply manifest.json:65]: Yes @victorhmp . Done :)

**Review by victorhmp — CHANGES_REQUESTED** [review]
The code LGTM, just please change back the version number in the manifest file so that we can merge this

File: manifest.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "store-theme",
-  "version": "4.3.0",
+  "version": "4.3.1",
```
[the remark this answers is not in the document]
Comment (carlosviniciusananias) [reply manifest.json]: @victorhmp Done

**Review by victorhmp — APPROVED** [review]

**Review by icazevedo — APPROVED** [review]

**Pull request merged**